packages/eslint-config/.eslintrc.js

```diff
@@ -33,7 +34,7 @@ module.exports = {
     "react/default-props-match-prop-types": 1,
     "react/prefer-stateless-function": 1,
     "react/no-unused-state": 1,
-    "react/prop-types": 1,
+    "react/prop-types": ["warn", { skipUndeclared: true }],
```

Comment:
This is a slight lint change, but one that reflects our usage in stellar.org. It will only lint for proptypes if some proptypes are assigned to the component. This could lead to some missing prop types, but we don't rely on them heavily anyway. Open to backing it out if people prefer.

Reply:
I think I would prefer that all proptypes to be properly defined; what's the advantage of doing this, besides not having to type out proptypes?

Reply:
Basically just that it's opt-in. This is our main eslint opt-out in stellar.org, but maybe it's an unusual case because of how a lot of the site is being rendered. Lots of trivial wrappers for e.g. MDX renderer. I'll back it out, I don't feel strongly about changing this.

Reply:
Stuff like this https://github.com/stellar/stellar.org/blob/master/src/templates/ApiReference.js#L183-L223